x-pack/docs/en/rest-api/security/put-app-privileges.asciidoc

```diff
@@ -0,0 +1,194 @@
+[role="xpack"]
+[[security-api-put-privileges]]
+=== Add application privileges API
+
+Creates or updates 
+{stack-ov}/security-privileges.html#application-privileges[application privileges].
+
+==== Request
+
+`POST /_xpack/security/privilege` +
+
+`PUT /_xpack/security/privilege`
+
+
+==== Description
+
+This API creates or updates privileges. To remove privileges, use the 
+<<security-api-delete-privilege,delete application privilege API>>. 
+
+For more information, see 
+{stack-ov}/defining-roles.html#roles-application-priv[Application privileges].
+
+To check a user's application privileges, use the
+<<security-api-has-privileges,has privileges API>>.
+
+////
```

Comment:
Let's just delete this commented out part now that the support for this has been removed